src/main/java/uk/gov/di/ipv/atp/dcs/utils/InstantConverter.java

```diff
@@ -0,0 +1,23 @@
+package uk.gov.di.ipv.atp.dcs.utils;
+
+
+import com.google.gson.*;
```

Comment:
We should avoid wildcard imports.

https://gds-way.cloudapps.digital/manuals/programming-languages/java.html#imports

Reply:
🙇🏻‍♂️